### PR TITLE
Scope instrumentation

### PR DESCRIPTION
**Pull request by fshot**

### CODE REVIEW

File: pkg/base/base_test.go [hunk 1]
```diff
@@ -120,10 +123,11 @@ spec:
 			assert.Equal(t, test.expected, string(b.Content))
 		})
 	}
-
 }
 
 func Test_ShouldBeIncludedInBaseKustomization(t *testing.T) {
+	test := scopeagent.StartTest(t)
+	defer test.End()
```
Comment (laverya) [inline pkg/base/base_test.go]: this is also instrumented within the test case loop

File: pkg/midstream/lists_test.go [hunk 2]
```diff
@@ -4,9 +4,12 @@ import (
 	"testing"
 
 	"github.com/stretchr/testify/assert"
+	"go.undefinedlabs.com/scopeagent"
 )
 
 func Test_findNewStrings(t *testing.T) {
+	scopetest := scopeagent.StartTest(t)
+	defer scopetest.End()
```
Comment (laverya) [inline pkg/midstream/lists_test.go]: move to inside the test case loop?

File: pkg/base/base_test.go (hunk 1 above)
Comment (fshot) [reply pkg/base/base_test.go]: thanks, fixed

File: pkg/midstream/lists_test.go (hunk 2 above)
Comment (fshot) [reply pkg/midstream/lists_test.go]: good catch, thanks

**Pull request merged**